client/consensus/manual-seal/src/rpc.rs

```diff
@@ -62,7 +62,7 @@ pub enum EngineCommand<Hash> {
 		/// sender to report errors/success to the rpc.
 		sender: Sender<()>,
 		/// finalization justification
-		justification: Option<Justification>,
+		justification: Option<EncodedJustification>,
```

Comment:
Does an `EncodedJustification` include the consensus engine id? We shouldn't be needing to tag with a consensus engine id here (only when persisting to client after).

Reply:
Yep that's right, this is effectively no change, it's still the same `Vec<u8>` as before